feat(related_issues): Initial documentation (#10419)
getsentry/sentry-docs: docs/product/issues/issue-details/index.mdx

A node will be red if it is associated with errors.
Click "View Full Trace" to display the [Trace View](/product/sentry-basics/tracing/trace-view).

### Related Issues

![Trace Related Issue](./img/trace_related_issue.png)

> Sometimes the Trace Navigator will be replaced by a Related Issue. A Related Issue is a different issue which occurred during the same trace associated to the event in view. This is useful for understanding the context of the current issue and may help you identify the root cause of the current issue. A Related Issue allows the developer to quickly navigate between the two related events.

Reviewer: ```suggestion
Sometimes the Trace Navigator will be replaced by a Related Issue. A Related Issue is a different issue that occurred during the same trace as the event in view. This is useful for understanding the context of the current issue and may help you identify its root cause. A Related Issue allows the developer to quickly navigate between the two related events.
```
